crossandeancommands/make_nexus.py

```diff
@@ -0,0 +1,82 @@
+"""
+Create a nexus file from the most recent data on Edictor.
+"""
+from pyedictor import fetch
+from lingpy import Wordlist
+from lingpy.convert.strings import write_nexus
+
+def run(args):
+
+    with open("./outputs/crossandean.tsv", "w",
```

Comment:
LingPy has a genuine but a bit complex function to load a wordlist from CLDF, so I'd suggest to use this one.

Reply:
Check here: http://lingpy.org/reference/lingpy.basic.html#lingpy.basic.wordlist.Wordlist.from_cldf

Reply:
If I choose to use this function, how do I access information such as "Cogids" and "Subgroup" that are stored in cognates.csv and languages.csv, respectively? Would I need to load them as a separate csv and make a lookup based on ID? The default of from_cldf, if I understand it right, would be to only load data from forms.csv.

Reply:
this is added with the next commit, including the data from `language_` and `cogid_`